### PR TITLE
Add section about AI connector to serverless

### PR DESCRIPTION
**Pull request by dedemorton**
Ports https://github.com/elastic/observability-docs/pull/3906 to serverless.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _co-preview-docs_ successfully!

* https://docs-elastic-4p4qydg6o-elastic-dev.vercel.app

* Built with commit 446c8262747dd84501deb427ade198384af99df1

> Issues? Visit #next-docs in Slack

File: docs/en/serverless/ai-assistant/ai-assistant.mdx [hunk 1]
```diff
@@ -240,12 +240,69 @@ You can continue a conversation from a contextual prompt by clicking **Start cha
 
 ![Observability AI assistant example](../images/ai-assistant-logs.png)
 
+[discrete]
+[[obs-ai-connector]]
+=== Add the AI Assistant connector to alerting workflows
```
Comment (dedemorton) [inline docs/en/serverless/ai-assistant/ai-assistant.mdx:245]:
```suggestion
### Add the AI Assistant connector to alerting workflows
```

File: docs/en/serverless/ai-assistant/ai-assistant.mdx [hunk 2]
```diff
@@ -240,12 +240,69 @@ You can continue a conversation from a contextual prompt by clicking **Start cha
...
+If this happens, modify the message specified in the connector configuration to avoid exceeding limits.
+
+When asked to send a message to another connector, such as Slack,
+the AI Assistant attempts to include a link to the generated conversation.
+
+![Message sent by Slack by the AI Assistant includes a link to the conversation](../images/obs-ai-assistant-slack-message.png)
+
+The Observability AI Assistant connector is called when the alert fires and when it recovers.
+
+To learn more about alerting, actions, and connectors, refer to <DocLink id="serverlessObservabilityAlerting" />
```
Comment (dedemorton) [inline docs/en/serverless/ai-assistant/ai-assistant.mdx:294]:
```suggestion
To learn more about alerting, actions, and connectors, refer to <DocLink id="serverlessObservabilityAlerting" />.
```

File: docs/en/serverless/ai-assistant/ai-assistant.mdx [hunk 3]
```diff
@@ -240,12 +240,67 @@ You can continue a conversation from a contextual prompt by clicking **Start cha
...
 
+<div id="token-limits"></div>
+
 ### Token limits
 
 Most LLMs have a set number of tokens they can manage in single a conversation.
 When you reach the token limit, the LLM will throw an error, and Elastic will display a "Token limit reached" error.
 The exact number of tokens that the LLM can support depends on the LLM provider and model you're using.
+If you are using an OpenAI connector, you can monitor token usage in **OpenAI Token Usage** dashboard.
+For more information, refer to the [OpenAI Connector documentation](((kibana-ref))/openai-action-type.html#openai-connector-token-dashboard)
```
Comment (mdbirnstiehl) [inline docs/en/serverless/ai-assistant/ai-assistant.mdx:304]:
```suggestion
For more information, refer to the [OpenAI Connector documentation](((kibana-ref))/openai-action-type.html#openai-connector-token-dashboard).
```

**Review by mdbirnstiehl — COMMENTED** [review]
LGTM except one tiny little dot at the end.

**Review by mdbirnstiehl — APPROVED** [review]
🚀

**Pull request merged**